CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 
 - feat: Add logic to add @sentry/nextjs if it's missing when running the wizard (#219)
 - fix: Print localhost with `http` instead of `https` (#212)
+- feat: Add project_platform as query param if -s and -i are set (#221)
```

Comment:
A more general question. Why is redirecting to the login not the default behavior?

Reply:
Ah I confused singup for login. Gotcha.